chromium_src/chrome/browser/mac/keystone_glue.mm

```diff
@@ -0,0 +1,29 @@
+#if defined(ENABLE_SPARKLE)
+#include "brave/browser/mac/sparkle_glue.h"
+#endif
+
+#define KeystoneEnabled KeystoneEnabled_ChromiumImpl
+#define CurrentlyInstalledVersion CurrentlyInstalledVersion_ChromiumImpl
+#include "../../../../../chrome/browser/mac/keystone_glue.mm"  // NOLINT
+#undef KeystoneEnabled
+#undef CurrentlyInstalledVersion
+
+namespace keystone_glue {
+
+bool KeystoneEnabled() {
+#if defined(ENABLE_SPARKLE)
```

Comment:
`ENABLE_SPARKLE` define is not existed in `chromium_src`.

Reply:
It works so must be an accident of some other dependency, but probably better to make it a buildflag so I'll update it